target/linux/ramips/dts/mt7620n_sunvalley.dtsi

```diff
@@ -1,30 +1,16 @@
-/dts-v1/;
-
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
```

Comment:
an empty line between license comment and #include

Reply:
Sure, done.